spec/query_spec.rb

```diff
@@ -70,6 +70,60 @@
     end
   end
 
+  describe "#start_time=" do
+    before do
+      subject.pattern    = "semla"
+      subject.start_time = time
+    end
+
```

Comment:
should we also include tests like this? `expect(subject.start_time.utc?).to be(true)` or is it too redundant?